packages/components/src/confirm-dialog/component.tsx

```diff
@@ -23,10 +23,10 @@ import { VStack } from '../v-stack';
 import * as styles from './styles';
 import { useCx } from '../utils/hooks/use-cx';
 
-function ConfirmDialog(
-	props: WordPressComponentProps< OwnProps, 'div', false >,
```

Comment:
What is the reason for removing the `false` argument ?

Reply:
Like I said was just messing around with different combinations to see if I could get the connectedProps to drop out correctly. Reverted.